dcrjson/dcrwalletextcmds.go

```diff
@@ -294,6 +294,47 @@ func NewPurchaseTicketCmd(fromAccount string, spendLimit float64, minConf *int,
 	}
 }
 
+// StartAutoBuyerCmd is a type handling custom marshaling and
+// unmarshaling of startautobuyer JSON RPC commands.
+type StartAutoBuyerCmd struct {
+	Passphrase        []byte
+	Account           float64
+	BalanceToMaintain *int64
+	MaxFeePerKb       *int64
+	MaxPriceRelative  *float64
+	MaxPriceAbsolute  *int64
+	VotingAddress     *string
+	PoolAddres        *string
+	PoolFees          *float64
+	MaxPerBlock       *int64
+}
+
+// NewStartAutoBuyerCmd creates a new StartAutoBuyerCmd.
+func NewStartAutoBuyerCmd(passphrase *[]byte, account float64, balanceToMaintain *int64, maxFeePerKb *int64, maxPriceRelative *float64, maxPriceAbsolute *int64, votingAddress *string, poolAddress *string, poolFees *float64,
```

Comment:
The first parameter shouldn't be a pointer to a `[]byte` since it's not optional and slice is already a reference type anyways.